src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__teacher_development.sql

```diff
@@ -0,0 +1,50 @@
+select
+    td.employee_number,
+    td.observer_employee_number,
+    td.observation_id,
+    td.rubric_name,
+    td.observation_score,
+    td.observed_at,
+    td.academic_year,
+    td.observation_type,
+    td.observation_type_abbreviation,
+    td.observation_subject,
+    td.observation_grade,
+    td.row_score,
+    td.measurement_name,
+    td.strand_name,
+    td.text_box,
+    td.growth_area,
+    td.glow_area,
+
+    srh.preferred_name_lastfirst as teammate,
+    srh.business_unit_home_name as entity,
+    srh.home_work_location_name as `location`,
+    srh.home_work_location_grade_band as grade_band,
+    srh.department_home_name as department,
+    srh.primary_grade_level_taught as grade_taught,
+    srh.job_title,
+    srh.report_to_preferred_name_lastfirst as manager,
+    srh.worker_original_hire_date,
+    srh.assignment_status,
+    srh.sam_account_name,
+    srh.report_to_sam_account_name,
+
+    coalesce(sr.preferred_name_lastfirst, td.observer_name) as observer_name,
+    if(sr.department_home_name = 'Teacher Development', 'TDT', 'NTNC') as observer_team,
+
+    os.final_score as performance_management_final_score,
+    os.final_tier as performance_management_final_tier,
+from {{ ref("int_performance_management__teacher_development") }} as td
+left join
+    {{ ref("base_people__staff_roster_history") }} as srh
+    on td.employee_number = srh.employee_number
+    and cast(td.observed_at as timestamp)
```

Comment:
hey @cbini I'm trying to use the observed_at_timestamp field upstream in int_performance_management__teacher_development but when I run it without casting to timestamp I get an error on this join

Reply:
that column is a date on the source table, needed to be converted to timestamp anyway